RELEASING.md

```diff
@@ -9,31 +9,41 @@ want to run the integration tests to confirm your version will work, and also
 use the Maven release plugin to update the versions.
 
 1) Run the integration tests locally with whatever GCP project you have at hand with credentials.
-   * Create the *integration-test-config.json* Spydra configuration as instructed in README, for example:
-    ```json
-    {
-      "log_bucket": "varjo-testing-spydra-logs",
-      "region": "europe-west1"
-    }
-    ```
    * Have credentials json file available that you want to use for running the integration tests.
    * Run the integration tests:
     ```bash
-    env GOOGLE_APPLICATION_CREDENTIALS=/path/to/key/my-gcp-credentials-key.json mvn clean install \
-    -Dinit-action-uri=gs://spydra-init-actions/spydra -Dtest-configuration-dir=/path/to/dir/with/test-config/
+    env GOOGLE_APPLICATION_CREDENTIALS=/path/to/key/my-gcp-credentials-key.json mvn clean verify
     ```
 
 2) After you have confirmed the integration tests run successfully, run Maven prepare:
    ```bash
-   env GOOGLE_APPLICATION_CREDENTIALS=/path/to/key/my-gcp-credentials-key.json mvn release:prepare \
-   -Darguments="-DskipTests -Dinit-action-uri=gs://spydra-init-actions/spydra -Dtest-configuration-dir=/path/to/dir/with/test-config/"
+   env GOOGLE_APPLICATION_CREDENTIALS=/path/to/key/my-gcp-credentials-key.json mvn release:prepare
    ```
-   * Notice that we do not run the tests again with the second run, as we have skipTests defined.
-     Release plugin requires us to use the -Darguments parameter wrapping to pass the parameters
-     to the wrapped Maven call from inside release plugin.
+   * Release plugin requires us to use the -Darguments parameter wrapping to pass the parameters
+     to the wrapped Maven call from inside release plugin. If you customized the configuration for
+     integration tests specify a suitable `-Darguments`:
    * Select the next version number and just get the release done.
 
-3) Confirm that the actual release runs successfully in https://travis-ci.org/spotify/spydra
-    Notice that Travis runs two parallel jobs for the two release commits that were made.
+3) The maven release plugin is configured not to push automatically, an automatic push would result
+   in a build only on the tip. Push `HEAD^1`, `HEAD` and tags individually and confirm that the
+   actual release runs successfully in https://travis-ci.org/spotify/spydra. Notice that Travis
+   runs two parallel jobs for the two release commits that were made.
```

Comment:
judging by my experience if you push commits one after another Travis cancels the job started for the first commit and starts a new one for the second. So what I had to do is to push HEAD^ then wait until the build is done and jars are pushed, and only then push the head of the repo to build the -SNAPSHOT version.